### PR TITLE
Add a new theme: A-RSnippet

### PR DESCRIPTION
**Pull request by huyingjie**

### CODE REVIEW

File: source/_data/themes.yml [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+  preview: http://arsnippet.yingjiehu.com/
+  tags:
+    - responsive
+    - bootstrap
+    - google-analytics
+    - portfolio
+    - disqus
+    - syntax-highligting
+    - font-awesome
+    -
```
Comment (JLHwung) [inline source/_data/themes.yml]: is `-` duplicated?

**Comment by huyingjie** [comment]
I deleted the redundant hyphen.

**Comment by JLHwung** [comment]
Thank you so much for your contribution. Hexo can't be rock without contributors like you.

**Comment by huyingjie** [comment]
@JLHwung Thank you for your maintaining Hexo. I made the theme and took me several days. You spend much more time on this open project than me. Thank you for your effort, energy, time and dedication. Is this your opencollective page? https://opencollective.com/hexo
This is not shown on https://hexo.io/about/index.html

**Pull request merged**